test_http_request.py

```diff
@@ -0,0 +1,83 @@
+import asyncio
+import pathlib
+import ssl
+from contextlib import asynccontextmanager, contextmanager
+from dataclasses import dataclass
+from tempfile import TemporaryDirectory
+from typing import AsyncIterator, Iterator
+
+import aiohttp
+import pytest
+from uvicorn import Config, Server  # type: ignore[import]
+from xpresso import App, Path
+
+import truststore
+
+
+@dataclass
```

Comment:
I think we'd want to parse the response from mkcert to check if the CA was already installed, otherwise we'd break users that do have the CA installed.

Reply:
So yeah there is no `mkcert -check` command to check if the CA is installed.
Trying to install is problematic because on a system that is not running as root (e.g. MacOS) it prompts the user to unlock with Touch ID, I assume the same is true with Windows Hello on Windows.
I think the best thing to do is to skip the test if the user didn't install mkcert's CA already.